src/Symfony/Component/Console/Formatter/OutputFormatter.php

```diff
@@ -142,14 +125,23 @@ public function getStyle($name)
     }
 
     /**
-     * Formats a message according to the given styles.
-     *
-     * @param string $message The message to style
-     *
-     * @return string The styled message
+     * {@inheritdoc}
```

Comment:
that one is not possible: $width is not in the interface, thus should be documented here